### PR TITLE
[README] "Java 7" --> "Java 8" (Related #1170)

### PR DESCRIPTION
**Pull request by ardacebi**
Change Java 7 to Java 8 as #1170 is merged.

### CODE REVIEW

**Comment by jisqyv** [comment]
@ardacebi Actually, for our production builds with Java 8 we are using OpenJDK

**Comment by ardacebi** [comment]
@jisqyv Is that related to the change applied on line 15 or 31?

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Check out our open source [site](http://appinventor.mit.edu/appinventor-sources/
 This is a quick guide to get started with the sources. More detailed instructions can be found [here](https://docs.google.com/document/pub?id=1Xc9yt02x3BRoq5m1PJHBr81OOv69rEBy8LVG_84j9jc), a slide show can be seen [here](http://josmas.github.io/contributingToAppInventor2/#/), and all the [documentation](http://appinventor.mit.edu/appinventor-sources/#documentation) for the project is available in our [site](http://appinventor.mit.edu/appinventor-sources/).
 
 ### Dependencies
-You will need a full Java JDK (version 7, preferably from Oracle; JRE is not enough) and Python to compile and run the servers.
+You will need a full Java JDK (version 8, preferably from Oracle; JRE is not enough) and Python to compile and run the servers.
```
Comment (jisqyv) [inline README.md]: We now use OpenJDK

**Comment by jisqyv** [comment]
@ardacebi Line 31

**Comment by ardacebi** [comment]
Line 31 is reverted.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Check out our open source [site](http://appinventor.mit.edu/appinventor-sources/
 This is a quick guide to get started with the sources. More detailed instructions can be found [here](https://docs.google.com/document/pub?id=1Xc9yt02x3BRoq5m1PJHBr81OOv69rEBy8LVG_84j9jc), a slide show can be seen [here](http://josmas.github.io/contributingToAppInventor2/#/), and all the [documentation](http://appinventor.mit.edu/appinventor-sources/#documentation) for the project is available in our [site](http://appinventor.mit.edu/appinventor-sources/).
 
 ### Dependencies
-You will need a full Java JDK (version 8, preferably from Oracle; JRE is not enough) and Python to compile and run the servers.
```
Comment (jisqyv) [inline README.md]: How about saying "full Java JDK (version 8, OpenJDK preferred; JRE is not enough) and ..."

**Comment by jisqyv** [comment]
@ardacebi See line comment I just added (I'm likely to be off-line shortly and my battery is almost gone...)

**Comment by ardacebi** [comment]
@jisqyv Sorry for the delay, I applied the change you suggested.

**Pull request merged**